app/view_models/paye/PayeIncomeTaxAndNics.scala

```diff
@@ -23,34 +23,46 @@ import view_models.{Amount, Rate}
 
 case class PayeIncomeTaxAndNics(taxYear: Int,
                                 scottishTaxBands: List[TaxBand],
-                                totalScottishIncomeTax: Amount) extends TaxYearFormatting
+                                ukTaxBands: List[TaxBand],
+                                totalScottishIncomeTax: Amount,
+                                totalUKIncomeTax: Amount) extends TaxYearFormatting
 
 object PayeIncomeTaxAndNics {
 
   lazy val scottishRates: List[String] = Play.current.injector.instanceOf[PayeConfig].scottishTaxBandKeys
 
+  lazy val uKRates: List[String] = Play.current.injector.instanceOf[PayeConfig].ukTaxBandKeys
+
   def apply(payeAtsData: PayeAtsData): PayeIncomeTaxAndNics = {
 
-    val scottishTaxBands = (for {
+    PayeIncomeTaxAndNics(payeAtsData.taxYear,
+      getTaxBands(payeAtsData,scottishRates),
+      getTaxBands(payeAtsData,uKRates),
+      getTotalIncomeTax(payeAtsData , "scottish_total_tax"),
+      getTotalIncomeTax(payeAtsData , "total_UK_income_tax"))
```

Comment:
Just want to double check 

> UK

is OK to be capitalised here as everything else is lower case

Reply:
agreed
changed to lower case